xtask/src/main.rs

```diff
@@ -99,7 +99,7 @@ fn wasm_opt(sh: &Shell) -> Result<()> {
     wasm(sh)?;
     cmd!(
 			sh,
-			"wasm-opt -Os --signext-lowering target/wasm32-unknown-unknown/release/seda_contract.wasm -o target/seda_contract.wasm"
+			"wasm-opt -Os --signext-lowering --enable-bulk-memory target/wasm32-unknown-unknown/release/seda_contract.wasm -o target/seda_contract.wasm"
```

Comment:
Just curious: reason for adding this?

Reply:
https://github.com/sedaprotocol/seda-chain-contracts/actions/runs/15164289057/job/42637909631

Reply:
Actually we reverted this change and hard pinned our rust version in the CI. As even with that option it was just broken.